be/CMakeLists.txt

```
@@ -264,7 +264,7 @@ check_function_exists(sched_getcpu HAVE_SCHED_GETCPU)
 #  -fno-omit-frame-pointers: Keep frame pointer for functions in register
 set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wall -Wno-sign-compare -Wno-unknown-pragmas -pthread")
 set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -fno-strict-aliasing -fno-omit-frame-pointer")
-set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -std=gnu++11 -D__STDC_FORMAT_MACROS")
+set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -std=gnu++17 -D__STDC_FORMAT_MACROS")
```

```suggestion
set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -std=c++17 -D__STDC_FORMAT_MACROS")
```
why use gnu++17 rather than c++17

---

C++ 17 is a subset of GNU++ 17. Gun extends some features not supported by standard C++, such as ```__ Int128``` which we used in Doris code. so I use GNU here.

---

OK